### PR TITLE
admin can add, update, delete user from frontend

### PR DESCRIPTION
**Pull request by MichelleCCodes**
## Description
- admin can create, edit, delete user

Fixes # (issue)
- CRUD operations are successful locally, but will need to check Okta to verify that CRUD is completed in that db

## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code

### CODE REVIEW

**Review by JonathanMary — APPROVED** [review]

File: src/components/pages/AdminTools/ManageUsers.js [hunk 1]
```diff
@@ -47,6 +47,7 @@ const ManageUsersPage = props => {
       .then(res => {
         alert(`${profile.first_name}'s profile was deleted`);
         console.log(res.data);
+        window.location.reload();
```
Comment (WeiWu2) [inline src/components/pages/AdminTools/ManageUsers.js:50]: Doesn't this reload the whole page? Is there a reason for using this? If you just wanted to re-render the page with updated user list I would imagine you just need to call the axios get profiles to set state and trigger a re-render

**Review by WeiWu2 — CHANGES_REQUESTED** [review]
Aside from the question about reloading, I would suggest replacing the regular axios calls with axiosWithAuth to keep it consistent and remove some console logs (unless your still testing with them)

File: src/components/pages/AdminTools/ManageUsers.js (hunk 1 above)
Comment (katieolson84) [reply src/components/pages/AdminTools/ManageUsers.js:50]: I agree. I think this was in the existing code base so it was getting reused but we should switch this over so the page does not rerender.

**Review by paulstgermain — CHANGES_REQUESTED** [review]
I like this change for the additional functionality and cleanliness of the code, but I'd request the same change as the others - change line 50 of ManageUsers.js to not reload the page. I see another pull request that implements axiosWithAuth, so after changing that reload line this pull request is ready for approval!

**Review by WeiWu2 — APPROVED** [review]

**Comment by JonathanMary** [comment]
There's merge conflicts now.  :(

**Pull request merged**